File: src/main/java/com/bmo/HelloWorld.java
package com.bmo;

public class HelloWorld {
    public static void main(String[] args) {
//        System.out.println("Hello");
        Person p = new Person();
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());
        /*int age = 19;
        Integer age2 = 19;
        char c = 'A';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        double height = 1.7;
        boolean adult = true;
        boolean enroll = false;
        String name = "Bmo";*/
    }
}

File: src/main/kotlin/com/bmo/HelloWorld.kt
package com.bmo

import javafx.scene.text.FontWeight

fun main() {
//    println("Hello Kotlin");
//    Human().hello();
    val h = Human(66.5f, 1.7f);
    println(h.bmi());
    /*val age : Int = 19;
    var weight = 66.5f;
    val name : String;
    name = "Bmo";*/
}

class Human(var weight: Float, var height : Float) {
    fun bmi() : Float {
        val bmi = weight / (height*height);
        return bmi;
    }

    fun hello(){
        println("Hello Kotlin");
    }
}